packages/@react-spectrum/s2/src/Accordion.tsx

```diff
@@ -0,0 +1,368 @@
+          regular: sizeValue(9),
+          spacious: sizeValue(13)
+        }
+      }
+    }
+  },
+  gap: {
+    size: {
+      S: sizeValue(7),
+      M: sizeValue(11),
+      L: sizeValue(14),
+      XL: sizeValue(17)
+    }
+  },
+  height: '[inherit]',
+  minHeight: 'control',
+  width: 'full',
+  backgroundColor: {
+    default: 'transparent',
+    isFocusVisible: 'transparent-black-100',
+    isHovered: 'transparent-black-100'
+  },
+  borderWidth: 0,
+  borderRadius: {
+    // Only rounded for keyboard focus and quiet hover.
+    default: 'none',
+    size: {
+      S: {
+        isFocusVisible: '[7px]',
+        isQuiet: {
+          isHovered: '[7px]',
+          isFocusVisible: '[7px]'
+        }
+      },
+      M: {
+        isFocusVisible: 'default',
+        isQuiet: {
+          isHovered: 'default',
+          isFocusVisible: 'default'
+        }
+      },
+      L: {
+        isFocusVisible: '[9px]',
+        isQuiet: {
+          isHovered: '[9px]',
+          isFocusVisible: '[9px]'
+        }
+      },
+      XL: {
+        isFocusVisible: 'lg',
+        isQuiet: {
+          isHovered: 'lg',
+          isFocusVisible: 'lg'
+        }
+      }
+    }
+  },
+  textAlign: 'start'
+});
+
+const chevronStyles = style({
+  rotate: {
+    deg: 0,
+    isExpanded: 90,
+    isRTL: {
+      deg: 180,
+      isExpanded: 270
+    }
+  },
+  transitionDuration: '100ms',
+  transitionProperty: 'rotate',
+  '--iconPrimary': {
+    type: 'fill',
+    value: 'currentColor'
+  },
+  flexShrink: 0
+});
+
+function AccordionHeader(props: AccordionHeaderProps, ref: DOMRef<HTMLDivElement>) {
+  let {
+    UNSAFE_style,
+    UNSAFE_className = '',
+    ...otherProps
+  } = props;
+  const domProps = filterDOMProps(otherProps);
+  let {direction} = useLocale();
+  let {isExpanded} = useContext(DisclosureStateContext)!;
+  let {size, density, isQuiet} = useContext(InternalAccordionGroupContext);
+  let isRTL = direction === 'rtl';
+  return (
+    <Header
+      {...domProps}
+      ref={ref}
+      style={UNSAFE_style}
+      className={(UNSAFE_className ?? undefined)}>
+      <Heading className={headingStyle}>
+        <Button className={({isHovered, isFocused, isFocusVisible, isDisabled}) => buttonStyles({size, isHovered, isFocused, isFocusVisible, density, isQuiet, isDisabled})} slot="trigger">
+          <Chevron size={size} className={chevronStyles({isExpanded, isRTL})} aria-hidden="true" />
+          {props.children}
+        </Button>
+      </Heading>
+    </Header>
+  );
+}
+
+/**
+ * A header for an accordion item. Contains a heading and a trigger button to expand/collapse the panel.
+ */
+let _AccordionHeader = /*#__PURE__*/ (forwardRef as forwardRefType)(AccordionHeader);
+export {_AccordionHeader as AccordionHeader};
+
+export interface AccordionPanelProps extends RACAccordionPanelProps, UnsafeStyles, DOMProps, AriaLabelingProps {
+  children: React.ReactNode
+}
+
+const panelStyles = style({
+  font: 'body',
+  height: 'auto',
```

Comment:
should be the default and is unneeded

Reply:
It was overflowing originally, but looks like its fixed now.